core/common/src/main/java/alluxio/wire/TieredIdentity.java

```diff
@@ -186,6 +188,13 @@ public static LocalityTier fromThrift(alluxio.thrift.LocalityTier localityTier)
       return new LocalityTier(localityTier.getTierName(), localityTier.getValue());
     }
 
+    /**
+     * Equality comparison for wire type locality tiers, two locality tiers are considered
+     * equal if both name and values are equal.
```

Comment:
or for the "node" tier, if the node names resolve to the same IP address

Reply:
Done